### ISSUE TITLE
device doesn't support speech input

### ISSUE THREAD

**Issue by Make42** [seymour-bootay/Mycroft-Android#16]
From https://github.com/seymour-bootay/Mycroft-Android/releases/tag/1.0.2-SNAPSHOT I downlosded the APK, but once Starter Mycroft says "Sorry, but your device doesn't support speech input". I have a Nexus 6P with Copperhead OS.

**Comment by seymour-bootay** [comment]
currently on Android OS is supported, I am not familiar with Copperhead OS does it have the google play services?

**Comment by BenjaminHCCarr** [comment]
@seymour-bootay Copperhead OS is a security enhanced fork. Play Services are optional.

**Comment by Make42** [comment]
@BenjaminHCCarr That is right. I do not have Play services though. Are they required for Mycroft?

**Comment by seymour-bootay** [comment]
currently the app is using Google's speech recognition so that is probably the issue.
